lib/ansible/modules/network/netact/netact_cm_command.py

```diff
@@ -0,0 +1,364 @@
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    # if the user is working with this module in only check mode we do not
+    # want to make any changes to the environment, just return the current
+    # state with no modifications
+    if module.check_mode:
+        return result
+
+    # manipulate or modify the state as needed (this is going to be the
+    # part where your module will do what it needs to do)
+
+    operation = module.params.get('operation')
+    if not operation:
+        module.fail_json(msg='Operation not defined', **result)
+
+    opsname = module.params.get('opsname')
+    dn = module.params.get('dn')
+    ws = module.params.get('ws')
+    mr = module.params.get('mr')
+
+    planname = module.params.get('planname')
+    typeoption = module.params.get('typeoption')
+    fileformat = module.params.get('fileformat')
+    filename = module.params.get('filename')
+
+    createBackupPlan = module.params.get('createBackupPlan')
+    backupPlanName = module.params.get('backupPlanName')
+    inputfile = module.params.get('inputFile')
+
+    extra_opts = module.params.get('extra_opts')
+    verbose = module.params.get('verbose')
+
+    # parameter checks
+
+    command = [racclimx, '-op', operation]
+
+    if opsname:
+        command.append('-opsName')
+        command.append(opsname)
+
+    if dn:
+        command.append('-DN')
+        command.append(dn)
+
+    if ws:
+        command.append('-WS')
+        command.append(dn)
+
+    if mr:
+        command.append('-MR')
+        command.append(dn)
+
+    if planname:
+        command.append('-planName')
+        command.append(planname)
+
+    if typeoption:
+        command.append('-type')
+        command.append(typeoption)
+
+    if fileformat:
+        command.append('-fileFormat')
+        command.append(fileformat)
+
+    if filename:
+        command.append('-fileName')
+        command.append(filename)
+
+    if createBackupPlan:
+        command.append('-createBackupPlan')
+        command.append(createBackupPlan)
+
+    if backupPlanName:
+        command.append('-backupPlanName')
+        command.append(backupPlanName)
+
+    if inputfile:
+        command.append('-inputFile')
+        command.append(inputfile)
+
+    if extra_opts:
+        command = command + extra_opts.split(" ")
+
+    if verbose:
+        if verbose == 'True':
+            command.append("-v")
+
+    rc, out, err = module.run_command(command, check_rc=True)
+    if rc != 0:
+        result['changed'] = False
+        module.fail_json(msg=err)
+    else:
+        result['changed'] = True
+        result['original_message'] = command
+        result['message'] = out
+
+    # in the event of a successful module execution, you will want to
+    # simple AnsibleModule.exit_json(), passing the key/value results
+    module.exit_json(**result)
+
+
+def main():
+    """
+    main
+    :return:
+    """
+    run_module()
```

Comment:
Maybe the `run_module` can be implemented in main() function iteself?

Reply:
Done